src/library/vicon_estimator.cpp

```diff
@@ -121,6 +143,45 @@ void TranslationalEstimator::setParameters(
   estimator_parameters_ = estimator_parameters;
 }
 
+//translational outlier rejection written by F.Grigis (fgrigis@student.ethz.ch) based on the code of Alex Millane
+bool TranslationalEstimator::detectTranslationalMeasurementOutlier(
```

Comment:
As above, I would recommend that you just call this method detectMeasurementOutlier. It's already in the TranslationalEstimator class.